instructions/create-aws-hosted-function.md

```diff
@@ -0,0 +1,69 @@
+# Build An AWS-Hosted Hello Buttons Skill
```

Comment:
is `AWS-Hosted` the official name we're going with "classic" skills?  It seems like the Alexa-hosted and AWS-hosted are essentially both hosted on AWS. I wonder if we should say "Build a Hello Buttons Skill Hosted in Your Own AWS Account"